src/services/dashboards/widgets/_base/base-pie/BasePieWidget.vue

```diff
@@ -114,6 +116,8 @@ const state = reactive({
     legends: computed<LegendConfig[]>(() => state.chartData.map((i) => ({
         name: i.provider,
     }))),
+    thisPage: 1,
+    limit: computed(() => state.thisPage > 3),
```

Comment:
`limit` is often used as `number` type in our project...so how about `disableNextPage` or else?